server/src/main/java/org/candlepin/dto/api/v1/ProductDTO.java

```diff
@@ -1035,6 +1035,8 @@ public boolean equals(Object obj) {
 
             // Check product content
             equals = equals && Util.collectionsAreEqual(this.getProductContent(), that.getProductContent());
+            equals = equals &&
+                Util.collectionsAreEqual(this.getProvidedProducts(), that.getProvidedProducts());
```

Comment:
Isn't the use of the `Util.collectionsAreEqual` unnecessary here and on all other DTOs (manifest ProductDTO, ProductData)? @Ceiu am I right in thinking we only need to use this method in hibernate entity classes, because hibernate collections don't implement .equals properly?

Reply:
Removed it.